packages/ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -76,6 +76,7 @@ export const TooltipContent: React.FC<
 			p="4"
 			borderRadius="6"
 			shadow="small"
+			wordBreak="break-word"
```

Comment:
padding here feels too little, is this per the designs? cc @julian-highlight 

![Screenshot from 2024-06-05 17-20-49](https://github.com/highlight/highlight/assets/1351531/f72420a3-639a-4d93-959f-ec2f631d6786)

Reply:
yes I think I need a few more pixels gap between rows, will fix